docs/validate.md

```diff
@@ -1,31 +1,304 @@
+    robot validate --input immune_exposures.owl \
+      --table immune_exposures.csv \
+      --reasoner hermit \
+      --no-fail true \
+      --format TXT \
+      --output-dir results/
 
----
+In this case the command will generate a single file called `immune_exposures.txt` in the `results/` directory.
+
+One can also specify multiple table files to validate against a single input ontology. In that case there will be multiple output files corresponding to each table in the output directory. For example:
+
+    robot validate --input immune_exposures.owl \
+      --table immune_exposures.csv \
+      --table immune_exposures_2.csv \
+      --reasoner hermit \
+      --no-fail true \
+      --format HTML \
+      --output-dir results/
+
+In this case two files: `immune_exposures.html` and `immune_exposures_2.html` will appear in the `results/` directory.
+
+### Formats
+
+* `txt`: a list of failed validations. E.g.:
+```
+At immune_exposures.csv row 17, column 2: Cell is empty but rule: "is-required true" does not allow this.
+```
+* `html`: a [Bootstrap](https://getbootstrap.com/) HTML version of the `--table`. Cells containing bad data (failed validations) are highlighted red. Hovering over the red cells shows a tooltip with the message. These tables use CSS and JavaScript plugins from the BootstrapCDN, therefore they require an internet connection to properly view the table. See [HTML Tables](#html-tables-standalone) for more details.
+* `xlsx`: an Excel spreadsheet version of the `--table`. Cells containing bad data are highlighted red and have a Comment on them containing the message.
+
+### Other options
+
+#### Exit Codes (`--no-fail`)
+
+If there are any invalid cells, `validate` will fail by default with exit code `1`. This is good for use in [`Makefile` workflows](/make), as it will stop the workflow when there is a non-zero exit code. You can override this with `--no-fail true` as shown in the above examples if you want to bypass failures and always exit with code `0`.
+
+#### Logging (`--silent`)
+
+`validate` will only print a summary message at the end if there were any failures (unless no `--format` is specified, in which case it will always print to STDOUT). If you would like to print all invalid data messages, include `--silent false`.
+
+#### Output Files (`--write-all`)
+
+`validate` will only write tables with failed validations to the output directory. If you wish to write _all_ tables, including those that did not have any failed validation, specify `--write-all true`.
+
+#### HTML Tables (`--standalone`)
+
+If the output format is HTML, all output tables will be written as "standalone" files. This means that they have a header containing the Bootstrap stylesheet and scripts ([for tooltips](https://getbootstrap.com/docs/4.5/components/tooltips)). If you want to plug the table data into an existing HTML file, you can use `--standalone true` to generate _just_ the table element.
```

Comment:
`--standalone true`  should mean a complete HTML file. `--standalone false` should mean just the HTML table. (This is the usage I've seen in lots of other tools, e.g. pandoc.) So this documentation is the opposite of what I would l like -- I'm not sure about the implementation.

Reply:
You're right. This is how it's implemented, the `--standalone true` in the docs was just a typo. I changed this to say `--standalone false`. The default behaviour is `true`.